### DISCUSSION TITLE
User Management for Prefect Server

### DISCUSSION THREAD

**Discussion by ChristianF88** [PrefectHQ/prefect#5922] [General]
Hi there,

I work in science and do have to write data pipelines all the time. One very import criterion to choose the right tool for the job is that fellow scientists must be able to interact with variables, setting etc. 

Comparing Airflow and Prefect, I know I want to use Prefect. It's great! Thanks for that! However, the self hosted usage is quite restricted.

Or at least I haven't seen any guide on how to set the server up, in a way that it has a more comprehensive set of features.

I was wondering whether the free open source version is planned to get user management as an included feature at some point?

Maybe with the new Iteration of Prefect, version 2.0?

Thank you very much!

Have a great one!
Christian

**Comment by taljaards** [comment]
Hi @ChristianF88 
You can refer to https://discourse.prefect.io/t/why-does-prefect-2-0-doesnt-include-auth-and-rbac-features-as-part-of-the-open-source-package/741

**Reply by ChristianF88** [reply]
Thank you very much! Do you know whether there're any efforts to incorporate these features becoming part of the open source version of Prefect?

**Comment by dinya** [comment]
Unfortunelly https://discourse.prefect.io/t/why-does-prefect-2-0-doesnt-include-auth-and-rbac-features-as-part-of-the-open-source-package/741 is broken link. The similar thread wasn't found among the discussions in the GitHub (wasn't exported from discourse platform?).

Dear team, could you please answer the question "Why does Prefect 2.0 doesn't include auth and rbac features as part of the open source package?" (and Prefect 3.x too at least in 3.3.x)?

Are there any plans to include these features in the open source version?

**Reply by cicdw** [reply]
There are no plans to include user management features in Prefect open source at this time. Prefect 3.0 does include _basic_ authentication but it's not user based.

**Reply by abarya** [reply]
Hey @cicdw, can you please point me to the docs for this basic auth you mentioned?

**Reply by cicdw** [reply]
Yup here you go: https://docs.prefect.io/v3/develop/settings-and-profiles#security-settings

**Reply by abarya** [reply]
Thanks, will try this.
